wear/src/main/java/com/android/wear/presentation/MainActivity.kt

```diff
@@ -0,0 +1,56 @@
+package com.android.wear.presentation
```

Comment:
Can you please refactor this package/folder to `com.android.gpstest.wear`?

Reply:
Ok, thanks.